local-terraform/main.tf

```diff
@@ -0,0 +1,31 @@
+terraform {
```

Comment:
if you'd like to keep this example of usage, can we move it into a new subdirectory under the examples directory: https://github.com/jdamata/terraform-provider-sonarqube/tree/master/examples